### ISSUE TITLE
Collaborating using GitHub

### ISSUE THREAD

**Issue by yiwen-h** [The-Strategy-Unit/data_science#212]
3 x 15 mins. Overall planning responsibility with @yiwen-h 

Thoughts below very much draft

What actually is GitHub anyway @yiwen-h - and how can it be useful for both coding- and non-coding people?

Working together with GitHub @StatsRhian and @Nat-Stephenson ?
- How to write a good issue
- How to review and tag issues sensibly
- Managing (merge) conflicts?

How to write a good PR (and do good code review): @matt-dray  and @craig-parylo ? #214

**Comment by yiwen-h** [comment]
same as #211 ?

**Comment by Nat-Stephenson** [comment]
I agree with @matt-dray regarding #211 as that seems something that maybe is slightly separate and would also make a very interesting topic seeing as there has been a lot of talk lately around the use of project management tools and what the Unit could/should be using. 

On another note, thank you @yiwen-h for setting out the initial structure of the session above, it flows well :)- are we wanting to do some kind of slide set? Or is it more talky and maybe some show and tell?

**Comment by Nat-Stephenson** [comment]
@StatsRhian and I haven't done any notes/slides as it is more of a chat format but I thought I would share our visual doodle instead :)

<img width="1920" height="1080" alt="Image" src="https://github.com/user-attachments/assets/28973ade-57bf-43e7-80d0-0e2db3434f30" />
